sensor_control/navigator_lidar_oa/src/lidar.cpp

```diff
@@ -386,163 +388,30 @@ int main(int argc, char* argv[])
 		if ( (ros::Time::now()-timer).toSec() > 600) { return -1; }
 		ros::Duration(0.1).sleep();
 	}
-	ROS_INFO_STREAM("ROS Master: " << ros::master::getHost());
+	ROS_INFO_STREAM("LIDAR | ROS Master: " << ros::master::getHost());
 
 	//Initialize Astar to use 8 way search method
 	astar.setMode(AStar::ASTAR,AStar::EIGHT);
 	
 	//Node handler
 	ros::NodeHandle nh;
 
-	//Setup action server
-	//MOVE_TO_SERVER actionServer(nh, "moveto", boost::bind(&actionExecute, _1, &actionServer) ,false);
-	// MOVE_TO_SERVER actionServer(nh, "moveto",actionExecute,false);
-	// actionServerPtr = &actionServer;
-	// actionServer.start();
-
 	//Subscribe to odom and the velodyne
 	ros::Subscriber sub1 = nh.subscribe("/velodyne_points", 1, cb_velodyne);
 	ros::Subscriber sub2 = nh.subscribe("/odom", 1, cb_odom);
 
 	//Publish occupancy grid and visualization markers
-	pubGrid = nh.advertise<nav_msgs::OccupancyGrid>("ogrid_batcave",10);
-	pubMarkers = nh.advertise<visualization_msgs::MarkerArray>("/unclassified/objects/markers",10);
-	pubBuoys = nh.advertise<navigator_msgs::PerceptionObjects>("/unclassified/objects",10);
+	pubGrid = nh.advertise<nav_msgs::OccupancyGrid>("ira_ogrid",10);
+	pubMarkers = nh.advertise<visualization_msgs::MarkerArray>("ira_markers",10);
+	pubCloudPersist = nh.advertise<sensor_msgs::PointCloud>("ira_persist",1);
+	pubCloudFrame = nh.advertise<sensor_msgs::PointCloud>("ira_frame",1);
+	pubCloudPCL = nh.advertise<sensor_msgs::PointCloud>("ira_pclcloud",1);
 
-	//Publish waypoints to controller
-	//pubTrajectory = nh.advertise<uf_common::PoseTwistStamped>("trajectory", 1);
-    //pubWaypoint = nh.advertise<PoseStamped>("waypoint", 1); //Do we need this?
+	//Service for object request
+	ros::ServiceServer service = nh.advertiseService("ira_database", objectRequest);
 
-	// pubMarkersSmall = nh.advertise<visualization_msgs::MarkerArray>("/vision/objects_unclassified",10);
 	//Give control to ROS
 	ros::spin();
```

Comment:
The mission system needs to get notified when a new object is added

Also, it will just be helpful in general to have a constantly updating list that any node can use. Right now I use it to tell Matt what the closest unknown object is so that we can classify 

It is possible to use the service call over and over again, but I think this will be a lot of extra work for the other nodes to implement.